### DISCUSSION TITLE
Session not Authorized for attachments

### DISCUSSION THREAD

**Discussion by Hedronmx** [Budibase/budibase#9022] [Help] [answered]
Hello! I am having problems with attachments as I cant have them set to public permission

**Comment by andz-bb** [comment]
hey @Hedronmx, can you give a bit more detail about how your screens and tables are set up? Alternatively if you send over an app export I'm happy to check things for you. You can send it directly to support@budibase.com or you can post it to me [on our Discord server here.](https://discord.gg/E3YXNAg4m5)

**Comment by melohagan** [comment, marked as answer]
Hey @Hedronmx 

This section in the docs shows you what you need to do to allow public attachment uploads: https://docs.budibase.com/docs/attachments#manage-access
